roxie/ccd/ccdserver.cpp

```diff
@@ -7238,6 +7238,7 @@ class CRoxieServerDedupKeepRightActivity : public CRoxieServerDedupActivity
 
     virtual void start(unsigned parentExtractSize, const byte *parentExtract, bool paused)
     {
+        assertex(numToKeep <= 1);
```

Comment:
This test is in the wrong place - the value will not be correctly initialised until *after* the base class start method is called.